### PR TITLE
Update recipe name of geno data submission recipe

### PR DESCRIPTION
**Pull request by proccaserra**

### CODE REVIEW

File: _toc.yml [hunk 1]
```diff
@@ -195,7 +195,7 @@
       title: Changing culture with the Dataset Maturity Model
       sections:
         - file: content/recipes/reusability/miappe #FCB061
-          title: Improving dataset maturity - the MIAPPE use case
+          title: MIAPPE-compliant plant genomic and genetic data submission to EMBL-EBI databases
```
Comment (proccaserra) [inline _toc.yml:198]:
```suggestion
          title: Improving dataset maturity - MIAPPE-compliant submission to EMBL-EBI databases
```

Shortening to accommodate UI constraints of the Table of Content side bar

File: content/recipes/reusability/miappe.md [hunk 2]
```diff
@@ -12,7 +12,7 @@
 :has_executable_code: nope
 :maturity_level: 3
 :maturity_indicator: 15,20
-:recipe_name: Improving dataset maturity - the MIAPPE use case
+:recipe_name: MIAPPE-compliant plant genomic and genetic data submission to EMBL-EBI databases
 ````
```
Comment (proccaserra) [inline content/recipes/reusability/miappe.md:16]:
```suggestion
          recipe_name: Improving dataset maturity - MIAPPE-compliant submission to EMBL-EBI databases
```

Shortening to accommodate UI constraints of the Table of Content side bar

**Review by proccaserra — COMMENTED** [review]
@erlefloch please see my suggestions

**Comment by erlefloch** [comment]
Thanks @proccaserra ! I applied your suggestions.

**Pull request merged**